server/src/testFixtures/java/org/elasticsearch/test/ESTestCase.java

```diff
@@ -613,6 +603,35 @@ public static double randomDouble() {
         return random().nextDouble();
     }
 
+    /**
+     * Returns a double value in the interval [start, end) if lowerInclusive is
+     * set to true, (start, end) otherwise.
+     *
+     * @param start          lower bound of interval to draw uniformly distributed random numbers from
+     * @param end            upper bound
+     * @param lowerInclusive whether or not to include lower end of the interval
+     */
+    public static double randomDoubleBetween(double start, double end, boolean lowerInclusive) {
```

Comment:
Could we re-use `RandomizedTest.biasedDoubleBetween` or is there a difference between the two?

Reply:
I think `RandomizedTest.biasedDoubleBetween` is more advanced specially as its biased, but this should be an improvement so I've pushed a fixup using this instead.